x/smart-account/authenticator/spend_limits_test.go

```diff
@@ -281,7 +281,7 @@ func (s *SpendLimitAuthenticatorTest) TestSpendLimit() {
 	s.Require().Contains(
 		err.Error(),
 		fmt.Sprintf(
-			"Current time %d.%d not within time limit None - %s.%s: execute wasm contract failed",
+			"Current time %d.%09d not within time limit None - %s.%s: execute wasm contract failed",
```

Comment:
wasn't this already merged? https://github.com/osmosis-labs/osmosis/pull/8120/files

Reply:
yeah not sure why it's in here.

Reply:
will revert the commit and let the merge figure out.